arlo-client/src/components/DataEntry/BallotAudit.tsx

```diff
@@ -127,34 +122,48 @@ const BallotAuditContest = ({
     })
   }
 
-  // Since the radio button values must be strings, not full objects,
+  // Since the checkbox button values must be strings, not full objects,
   // we condense our data model for interpretations to strings:
   // - For buttons representing interpretation VOTE (one for each
   // contest choice), the value is interpretation.choiceId
   // - For buttons representing other interpretations (BLANK,
   // CANT_AGREE), the value is interpretation.interpretation
-  const radioProps = {
-    name: `interpretation-${contest.name}`,
+  const checkboxProps = (value: string) => ({
+    name: `interpretation-${contest.name}-${value}`,
```

Comment:
wondering if the `name` should be the same for all checkboxes in a contest or if each one needs its own unique name. not quite sure what `name` is for

Reply:
@jonahkagan `name` is normally used in this context for Formik to know which element is which, and so unique names is more important there. But since we're setting the whole interpretation at once, it's not needed here now, so I'm removing it.